service/src/main/java/bio/terra/externalcreds/services/ProviderService.java

```diff
@@ -41,33 +21,53 @@
 @Slf4j
 public class ProviderService {
 
-  public static final String PASSPORT_JWT_V11_CLAIM = "passport_jwt_v11";
-  public static final String GA4GH_PASSPORT_V1_CLAIM = "ga4gh_passport_v1";
-  public static final String GA4GH_VISA_V1_CLAIM = "ga4gh_visa_v1";
-  public static final String VISA_TYPE_CLAIM = "type";
-  public static final String JKU_HEADER = "jku";
   public static final String EXTERNAL_USERID_ATTR = "email";
 
   private final ExternalCredsConfig externalCredsConfig;
   private final ProviderClientCache providerClientCache;
   private final OAuth2Service oAuth2Service;
   private final LinkedAccountService linkedAccountService;
+  private final LinkedAccountDAO linkedAccountDAO;
```

Comment:
`ProviderService` does not use any transactional annotations so should not access DAOs directly. Pipe calls to `linkedAccountDAO` through `linkedAccountService` with proper transactional annotations in `LinkedAccountService`